readme.txt

```diff
@@ -13,6 +13,13 @@ An open platform for the collaborative evaluation of knowledge.
 
 Hypothesis is a web annotation tool that allows users to provide commentary, references, and insight on top of news, blogs, scientific articles, books, terms of service, ballot initiatives, legislation and regulations, software code and more. You can find out more at [http://hypothes.is/](http://hypothes.is/)
 
+This plugin will allow you to automatically embed Hypothesis in your site, and integrate with the Hypothesis annotations API.
```

Comment:
What exactly does "and integrate with the Hypothesis annotations API" entail?

Reply:
I actually don't love this piece, so I'm open to suggestions.

The motivation for this is that we were told we had to explicitly document the fact that this plugin will cause a third party service to be consumed.

What I was trying is to somehow explain that when the plugin is enabled, it embeds hypothesis client script on the page, and that, in turn, consumes a third party service, which is what I called "Hypothesis annotations API".

Consuming third party services is allowed as long as it's clearly documented.

Reply:
OK. I think this is clearer with just the first part of the sentence - "This plugin will allow you to automatically embed Hypothesis in your site". This makes it sound like Disqus or some other widget that you can embed, which is accurate.

Reply:
Yep, agree. Hopefully they are not too picky with that 😅 

Actually, Disqus is a good example. I'm sure they have an official WordPress plugin and we can check what their readme file says

Reply:
From their readme https://wordpress.org/plugins/disqus-comment-system/

> The Disqus for WordPress plugin lets site owners and developers easily add Disqus to their sites, replacing the default WordPress comment system

So yeah, I think the first part should be enough.